roles/rsyslog/templates/71-additional-log-server.conf.j2

```diff
@@ -1 +1,5 @@
+{% if rsyslog_additional_protocol|default('udp') == 'udp' %}
 *.* @{{ rsyslog_additional_host }}:{{ rsyslog_additional_port }}
+{% else %}
+*.* @@{{ rsyslog_additional_host }}:{{ rsyslog_additional_port }}
```

Comment:
LGTM, but:

according to this [post](https://www.rsyslog.com/sending-messages-to-a-remote-syslog-server/) this is a legacy format that should not be used anymore.
It seems that the new format is more readable. so we might want to adapt this in the future.

Reply:
@artificial-intelligence Please open an issue for this task in osism/issues.

Reply:
https://github.com/osism/issues/issues/654